src/datachain/cli/parser/utils.py

```diff
@@ -1,11 +1,27 @@
-from argparse import Action, ArgumentParser, ArgumentTypeError
+from argparse import Action, ArgumentParser, ArgumentTypeError, HelpFormatter
 from typing import Union
 
 from datachain.cli.utils import CommaSeparatedArgs
 
 FIND_COLUMNS = ["du", "name", "path", "size", "type"]
 
 
+class CustomHelpFormatter(HelpFormatter):
+    def add_arguments(self, actions):
+        # Sort arguments to move --help and --version to the end
+        normal_actions = [
+            a for a in actions if a.dest not in ("help", "verbose", "quiet")
+        ]
+        special_actions = [a for a in actions if a.dest in ("help", "verbose", "quiet")]
+        super().add_arguments(normal_actions + special_actions)
+
+
+class CustomArgumentParser(ArgumentParser):
+    def error(self, message):
+        message = message.replace(" 'internal-run-udf', 'internal-run-udf-worker',", "")
```

Comment:
What do you think about moving these `["internal-run-udf", "internal-run-udf-worker"]` to `internal_commands` variable?